src/core/utils/detectScriptType.ts

```diff
@@ -9,7 +9,11 @@ export function detectScriptType() {
     if (window.location.pathname.includes('background')) return 'background';
     if (window.location.pathname.includes('contentscript'))
       return 'contentScript';
-    if (window.location.pathname.includes('popup')) return 'popup';
+    if (
+      window.location.pathname.includes('popup') &&
+      !window.location.origin.includes('trezor')
+    )
+      return 'popup';
```

Comment:
Trezor's flow contains a page with 'popup' in the pathname, so we needed to be more specific here.